docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: parent-child-accounts
+description: 'Learn how parent and child accounts can help Akamai partners manage multiple accounts.'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2024-04-23
+keywords: ["akamai partners", "parent", "child", "parent/child relationship"]
+modified_by:
+  name: Linode
+published: 2022-04-23
+title: Parent and Child Accounts for Akamai Partners
+aliases: ['/guides/akamai/get-started/parent-child-account/']
+authors: ["Linode"]
+promo_default: false
+---
+## Overview
+The parent and child accounts feature was designed with Akamai partners and their clients in mind. It allows partners to manage multiple end customers’ accounts with a single login to their company’s account.
+
+{{< note type="secondary" >}}
+This feature is available only for Akamai partners and their end customers with an Akamai Cloud Computing contract. To learn more about it, contact your Akamai representative.
+
+{{< /note >}}
+
+Depending on whether you're an Akamai partner or a partner’s end customer, this feature has different implications. Select your role to go to content that applies to you.
+- [I’m an Akamai partner](#for-akamai-partners).
+- [I’m a partner’s end customer](#for-end-customers).
+
+
+## For Akamai partners
+### About the feature
+
+The parent and child accounts feature allows you, an Akamai partner (parent account holder), to switch between and manage your end customers’ accounts (child accounts) in Cloud Manager.
+
+Child account users can monitor your actions on their account as all actions are logged and can be viewed in the Cloud Manager, either through the Events panel or the [Events](https://cloud.linode.com/events) page.
+
+The parent/child relationship doesn’t restrict the child account and its users from creating support tickets or signing up for managed services for their accounts.
+
+### Terminology
+
+#### Parent account
+In the context of your account type, you need to be familiar with the following terms.
```

Comment:
I'd simplify this to:

"With your account type, ensure you understand these terms:"